src/vsr/superblock.zig

```diff
@@ -177,6 +167,10 @@ pub const SuperBlockHeader = extern struct {
                     .free_set_last_block_checksum = 0,
                     .free_set_last_block_address = 0,
                     .free_set_size = 0,
+                    .client_sessions_checksum = vsr.checksum(&.{}),
```

Comment:
While we are here, might do `comptime vsr.checksum(&.{})`. Not that it _really_ matters when we
compute the empty hash here, but computing it at runtime generally irks me a bit, as that's not a
trivial operation.